### PR TITLE
feat: add principles page

### PR DESCRIPTION
**Pull request by braaar**
we should merge #332 and rebase/merge this before adding a link to `/principles` in navbar. It is linked from the main page "Les mer om våre prinsipper"

Screenshot 2022-02-11 at 11 26 15

localhost_3000_principles(iPhone XR)

### CODE REVIEW

**Review by cobraz — COMMENTED** [review]
Seems like there's illustrations that are no longer needed.

**Comment by braaar** [comment]
Is `lead` a better name than `headline` for this text?

File: src/theme.tsx [hunk 1]
```diff
@@ -83,10 +83,15 @@ export const theme = merge(webTheme, {
       lineHeight: 0.9,
       fontWeight: 800,
     },
+    heading: { fontWeight: 800 },
     subtitle: {
       color: 'green120',
       fontWeight: 'medium',
     },
+    headline: {
```
Comment (braaar) [inline src/theme.tsx:91]:
```suggestion
    lead: {
```

File: src/pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { Grid, Heading, Link, Paragraph } from 'theme-ui';
+import { Layout } from '../components/layout/layout';
+
+const About: React.FC = () => {
+  return (
+    <Layout>
+      <Grid sx={{ gap: 3 }}>
+        <Heading variant="title">Våre prinsipper</Heading>
+        <Paragraph variant="headline">
```
Comment (braaar) [inline src/pages/about.tsx:9]:
```suggestion
        <Paragraph variant="lead">
```

File: src/components/feature-box.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Box, Flex } from 'theme-ui';
```
Comment (cobraz) [inline src/components/feature-box.tsx:2]: This looks unused.

**Comment by braaar** [comment]
Should this page be under `/principles` instead of `/about`?

**Review by cobraz — APPROVED** [review]

**Comment by cobraz** [comment]
> Should this page be under `/principles` instead of `/about`?

Let's do it!

**Pull request merged**